src/Service/FormAnnotationBuilderFactory.php

```diff
@@ -96,7 +96,7 @@ private function injectFactory(
         AnnotationBuilder $annotationBuilder
     ) {
         if ($formElementManager instanceof FormElementManagerV2Polyfill) {
-            $formElementManager->injectFactory($annotationBuilder, $container);
+            $formElementManager->injectFactory($annotationBuilder, $formElementManager);
```

Comment:
Why would the formElementManager inject itself? I still dont get the meaning of this.